### PR TITLE
London Class 7 - Askin Ekinci - JavaScript Core 3 - Week 1

### PR DESCRIPTION
**Pull request by askinekincihf**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Askin Ekinci
- Your City: London-7
- Your Slack Name: Askin Ekinci

# Homework Details

- Module: JavaScript Core-3
- Week: 1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: mandatory/debugging-practice/script.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ window.addEventListener("load", function (e) {
 
 function populateStorage() {
   if (myLibrary.length == 0) {
-    let book1 = new Book("Robison Crusoe", "Daniel Defoe", "252", true);
+    let book1 = new Book("Robinson Crusoe", "Daniel Defoe", "252", true);
     let book2 = new Book(
```
Comment (DanGhanbari) [inline mandatory/debugging-practice/script.js:11]: Good job for finding a typo!

File: mandatory/debugging-practice/script.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ function submit() {
     alert("Please fill all fields!");
```
Comment (DanGhanbari) [inline mandatory/debugging-practice/script.js:37]: Askin do you think if you leave author input blank, you get an alert?!

File: mandatory/debugging-practice/script.js (hunk 1 above)
Comment (askinekincihf) [reply mandatory/debugging-practice/script.js:11]: Thanks Daniel for the feedback.

File: mandatory/debugging-practice/script.js (hunk 2 above)
Comment (askinekincihf) [reply mandatory/debugging-practice/script.js:37]: You are right Daniel. I didn't notice it . Fixed below.
```
author.value == null ||
author.value == "" ||
```

File: mandatory/debugging-practice/script.js [hunk 3]
```diff
@@ -37,8 +37,8 @@ function submit() {
...
     return false;
   } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+    let book = new Book(title.value, author.value, pages.value, check.checked); // Fixed author name.
+    myLibrary.push(book);
```
Comment (rfloresram) [inline mandatory/debugging-practice/script.js:43]: Interesting and synthesized way to enforce the function, definitely with you one learns to do things quickly and effectively, congratulations!

**Pull request closed without merging**